charts/reform-scan-blob-router/values.yaml

```diff
@@ -28,6 +28,7 @@ java:
     FLYWAY_SKIP_MIGRATIONS: true
 
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
+    HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
```

Comment:
Wasn't value different initially when I saw? So which one we are going for the main template :)

Reply:
yeah, following existing convention now. Prod values are set in flux.

Reply:
ah yea, of course :)